ethportal-peertest/src/jsonrpc.rs

```diff
@@ -221,40 +258,26 @@ pub async fn test_jsonrpc_endpoints_over_ipc(peertest_config: PeertestConfig, bu
             // break out of loop here since EOF is not sent, and loop will hang
             break;
         }
+        // Short sleep to make sure all peertest nodes can connect
+        thread::sleep(time::Duration::from_secs(1));
```

Comment:
Sleeps stand out as a red flag in tests. One second is pretty long to set as a minimum for all of these tests. Also, it seems likely to be a source of flakiness in the future.

Ideal would be to have some kind of trigger that allows the test to continue after nodes connect. Otherwise, it can be reasonable to poll for results and retry up to a maximum test length. Much better to have a single test take 3 seconds before failing than to have every passing test take 1 second.